format/hint/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "value": {
```

Comment:
I find the original `text` more accurate than `value`. Could the hint be something else than text? If so, maybe reusing the `content` schema would be better.